src/components/Nav.jsx

```diff
@@ -0,0 +1,36 @@
+import $ from 'jquery';
```

Comment:
I would reccomend doing this in a "react-y" way-- normally we never mix jquery and react. it will also be easier with react!